CoreScriptsRoot/Modules/BackpackScript.lua

```diff
@@ -1172,90 +1267,102 @@ end
-OpenInventoryButton.Position = UDim2.new(0.5, -22, 1, -120)
-OpenInventoryButton.Image = "rbxasset://textures/ui/Backpack/ScrollUpArrow.png";
-OpenInventoryButton.MouseButton1Click:connect(function()
-	BackpackScript.OpenClose()
-end)
+do
+	OpenInventoryButton.Size = UDim2.new(0, 30, 0, 30)
+	OpenInventoryButton.Position = UDim2.new(0.5, -15, 1, -110)
+	OpenInventoryButton.Image = "rbxasset://textures/ui/Backpack/ScrollUpArrow.png";
+	OpenInventoryButton.MouseButton1Click:connect(function()
+		BackpackScript.OpenClose()
+	end)
+	OpenInventoryButton.SelectionGained:connect(function()
+		OpenInventoryButton.ImageColor3 = ARROW_HOVER_COLOR
+	end)
+	OpenInventoryButton.SelectionLost:connect(function()
+		OpenInventoryButton.ImageColor3 = Color3.new(1,1,1)
+	end)
+	local openInventoryButtonSelectionObject = NewGui('Frame', 'Selection')
+	openInventoryButtonSelectionObject.Visible = false
+	OpenInventoryButton.SelectionImageObject = openInventoryButtonSelectionObject
+end
 
-CloseInventoryButton = NewGui('ImageButton', 'CloseInventory')
-CloseInventoryButton.Size = UDim2.new(0, 40, 0, 40)
+CloseInventoryButton = MakeVRRoundButton('CloseInventory', 'rbxasset://textures/ui/Keyboard/close_button_icon.png')
 CloseInventoryButton.Position = UDim2.new(0, 0, 0, -50)
-CloseInventoryButton.Image = "rbxasset://textures/ui/Keyboard/close_button_background.png";
-CloseInventoryButton.Visible = true
 CloseInventoryButton.MouseButton1Click:connect(function()
 	if InventoryFrame.Visible then
 		BackpackScript.OpenClose()
 	end
 end)
-do
-	CloseInventoryButtonSelectionObject = NewGui('ImageLabel', 'Selection')
-	CloseInventoryButtonSelectionObject.Size = UDim2.new(0.9,0,0.9,0);
-	CloseInventoryButtonSelectionObject.Position = UDim2.new(0.05,0,0.05,0);
-	CloseInventoryButtonSelectionObject.Image = "rbxasset://textures/ui/Keyboard/close_button_selection.png";
-	CloseInventoryButton.SelectionImageObject = CloseInventoryButtonSelectionObject
-
-	CloseInventoryButtonIcon = NewGui('ImageLabel', 'Icon')
-	CloseInventoryButtonIcon.Size = UDim2.new(0.5,0,0.5,0);
-	CloseInventoryButtonIcon.Position = UDim2.new(0.25,0,0.25,0);
-	CloseInventoryButtonIcon.Image = "rbxasset://textures/ui/Keyboard/close_button_icon.png";
-	CloseInventoryButtonIcon.Parent = CloseInventoryButton;
-end
 
 LeftBumperButton = NewGui('ImageLabel', 'LeftBumper')
 LeftBumperButton.Size = UDim2.new(0, 40, 0, 40)
 LeftBumperButton.Position = UDim2.new(0, -LeftBumperButton.Size.X.Offset, 0.5, -LeftBumperButton.Size.Y.Offset/2)
--- LeftBumperButton.Image = 'rbxasset://Textures/Sword128.png'
+LeftBumperButton.Image = "rbxasset://textures/ui/Backpack/ScrollUpArrow.png";
 
 RightBumperButton = NewGui('ImageLabel', 'RightBumper')
 RightBumperButton.Size = UDim2.new(0, 40, 0, 40)
 RightBumperButton.Position = UDim2.new(1, 0, 0.5, -RightBumperButton.Size.Y.Offset/2)
--- RightBumperButton.Image = 'rbxasset://Textures/Sword128.png'
+RightBumperButton.Image = "rbxasset://textures/ui/Backpack/ScrollUpArrow.png";
```

Comment:
We don't have art for these yet so they are actually not being shown. Eventually we will get new images for both left and right.